Sources/AWSLambdaRuntimeCore/Terminator.swift

```diff
@@ -137,3 +137,10 @@ extension LambdaTerminator {
         let underlying: [Error]
     }
 }
+
+// Ideally this would not be @unchecked Sendable, but Sendable checks do not understand locks
+// We can transition this to an actor once we drop support for older Swift versions
```

Comment:
We can transition this to an actor once we have custom executors :)

Reply:
that too